.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+# Description
+
+Please include a summary of the change and which issue is fixed (if relevant).
+
+Fixes # (issue)
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+# How to test this change?
+
+Please describe the steps to test your changes.
+
+- Step 1
+- Step 2
+- etc
+
+
+# Checklist:
+
+Please delete options that are not relevant.
+
+- [ ] My commit message follows [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
+- [ ] My code is formatted to [Go standards](https://go.dev/blog/gofmt)
+- [ ] I have performed a self-review of my own code
+- [ ] I have made corresponding changes to the documentation
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] New and existing unit tests pass locally with my changes
```

Comment:
Should we add the local test commands for this? `make build`

Reply:
I added a link to the testing section of the README.